_translator-files/po/de/Privacy-Statement.po

```diff
@@ -5,14 +5,16 @@
 #
```

Comment:
In general, I think the privacy page shouldn't be translated at all (to be on the safe side...)

Reply:
Would be ok for me.
Please check with the team.